mbuild/tests/test_xyz.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pytest
+
+import mbuild as mb
+from mbuild.tests.base_test import BaseTest
+
+
+class TestXYZNoTOP(BaseTest):
+    def test_load(self, ethane):
+        ethane.save(filename='ethane.xyz')
+        ethane.save(filename='ethane.pdb')
+        ethane_in = mb.load('ethane.xyz')
+        assert len(ethane_in.children) == 8
+        assert set([child.name for child in ethane_in.children]) == {'C', 'H'}
+
+class TestXYZWithTOP(BaseTest):
+    def test_load(self, ethane):
+        ethane.save(filename='ethane.xyz')
+        ethane.save(filename='ethane.pdb')
+        ethane_in = mb.load('ethane.xyz', top='ethane.pdb')
+        assert len(ethane_in.children) == 8
+        assert set([child.name for child in ethane_in.children]) == {'C', 'H'}
```

Comment:
Can you also assert that the correct number of bonds are present in this test, and that no bonds are present in the other test?